japicmp/src/main/java/japicmp/cli/JApiCli.java

```diff
@@ -69,6 +70,12 @@ private void generateOutput(Options options, List<JApiClass> jApiClasses, JarArc
 				throw new JApiCmpException(JApiCmpException.Reason.IoException, "Could not write HTML file: " + e.getMessage(), e);
             }
         }
+		if (options.isMarkdown()) {
+			MarkdownOutputGenerator markdownOutputGenerator = new MarkdownOutputGenerator(options, jApiClasses);
+			String output = markdownOutputGenerator.generate();
+			System.out.println(output);
+			return;
```

Comment:
I would remove the `return` and make an else case here, as otherwise the `IncompatibleErrorOutput` is not executed.